clang/lib/Basic/LangOptions.cpp

```diff
@@ -34,8 +34,8 @@ void LangOptions::resetNonModularOptions() {
   // invocations that cannot be round-tripped to arguments.
   // FIXME: we should derive this automatically from ImpliedBy in tablegen.
   AllowFPReassoc = UnsafeFPMath;
-  NoHonorNaNs = FiniteMathOnly;
-  NoHonorInfs = FiniteMathOnly;
+  NoHonorNaNs = 0;
+  NoHonorInfs = 0;
```

Comment:
I think we don't need this code at all because `LangOptions.def` defines both of these with a default value of 0 anyway, so the x macros above should already do this automatically, right?